Dockerfile

```diff
@@ -36,6 +36,20 @@ ENV PATH="/home/shardeum/.npm-global/bin:/home/shardeum/.cargo/bin:${PATH}"
 # Install global npm packages
 RUN npm install -g pm2
 
+RUN mkdir -p /home/shardeum/bin
+ENV PATH="/home/shardeum/bin:${PATH}"
+
+
+# Install litestream using pre-built binary
+RUN case $(uname -m) in \
```

Comment:
this comment is for this [line](https://github.com/shardeum/ldrpc-docker/pull/18/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R66)
Why have we only created shardeum/db and but not relayer-collector/db?